fe/fe-core/src/main/java/org/apache/doris/journal/JournalEntity.java

```diff
@@ -844,6 +846,21 @@ public void readFields(DataInput in) throws IOException {
                 isRead = true;
                 break;
             }
+            case OperationType.OP_UPDATE_TABLE_STATS: {
+                data = TableStatsMeta.read(in);
```

Comment:
If we write this new meta data , and downgrade to old version, it may cause problem because the old version can not recognize this kind of meta data.